modules/channel-vonage/src/senders/common.ts

```diff
@@ -40,8 +41,10 @@ export class VonageCommonSender implements ChannelSender<VonageContext> {
         )
       })
 
-      // TODO : delay for sandbox. don't delay when not in sandbox (should have a isSandbox property in context)
-      await Promise.delay(1000)
+      if (i !== 0 && context.isSandbox) {
```

Comment:
Why if `i !== 0` ? This means that there won't be any waiting between the first and the second message!

I think it would be simpler if we always wait 1sec after each call (even the last one):
```suggestion
      if (icontext.isSandbox) {
```

Reply:
I was trying to avoid having a 2 second wait caused by the typing delay put I just realized this solution actually doesn't make sense so I'll keep it as it was